packages/vscode-lit-plugin/package.json

```diff
@@ -25,31 +25,120 @@
 	"main": "./out/extension",
 	"scripts": {
 		"clean": "rimraf out",
-		"vscode:prepublish": "npm run compile",
-		"compile": "npm run clean & tsc -p ./",
 		"watch": "tsc -watch -p ./",
 		"publish": "vsce publish",
-		"package": "vsce package",
+		"package": "wireit",
 		"install:safe": "npm install --ignore-scripts --no-package-lock && npm i --no-package-lock",
 		"readme": "readme generate -i readme.blueprint.md -c readme.config.json",
-		"test": "npm run compile && node ./out/test/scripts/test-runner.js",
-		"test-packaged": "vsce package -o ./out/packaged.vsix && rm -rf ../../../packaged-extension/ && mkdir ../../../packaged-extension/ && unzip -qq ./out/packaged.vsix -d ../../../packaged-extension/ && node ./out/test/scripts/test-runner.js ../../../packaged-extension/extension/"
+		"test": "wireit",
+		"build": "wireit",
+		"test:normal": "wireit",
+		"test:packaged": "wireit",
+		"create-isolated-extension-dir": "wireit",
+		"test:packaged-sequentialized": "wireit"
+	},
+	"wireit": {
+		"build": {
+			"dependencies": [
+				"../ts-lit-plugin:build",
+				"../lit-analyzer:build",
+				"../../:copylink"
+			],
+			"command": "tsc --build --pretty",
+			"files": [
+				"src/**/*",
+				"tsconfig.json"
+			],
+			"output": [
+				"out/**/*.js",
+				"out/**/*.d.ts",
+				"out/**/*.d.ts.map",
+				".tsbuildinfo"
+			],
+			"clean": "if-file-deleted"
+		},
+		"test": {
+			"dependencies": [
+				"test:normal",
+				"test:packaged-sequentialized"
+			]
+		},
+		"test:normal": {
+			"dependencies": [
+				"build"
+			],
+			"files": [
+				"package.json",
+				"schemas/**/*"
+			],
+			"output": [],
+			"command": "node ./out/test/scripts/test-runner.js"
```

Comment:
Looks like this is a build artifact, so it doesn't need to be in `files`. (Just confirming?)

Same for the other scripts below.